design/kiota-e2e.md

```diff
@@ -23,8 +23,155 @@ Before we jump into the end-to-end walk-through, it's important to set some cons
 
 ## Table of Contents
 
--   [NodeJS e2e using the Service library](#node-service-library)
--   [NodeJS e2e using the Core library](#node-core-library)
+- [Foudation](#foundation)
+- [NodeJS e2e using the Service library](#node-service-library)
+- [NodeJS e2e using the Core library](#node-core-library)
+
+## <a id="foundation"></a> Foundation
```

Comment:
```suggestion
## Foundation
```

github adds the anchors automatically for you when using headings

Reply:
They don't allow me to "know" what will be the generated anchor. So I prefer this for more complex TOCs.

Reply:
it uses kebab casing Foundation => foundation, My Foundation => my-foundation